stdlib/private/SwiftPrivateLibcExtras/SwiftPrivateLibcExtras.swift

```diff
@@ -131,6 +131,8 @@ public func _stdlib_pipe() -> (readEnd: CInt, writeEnd: CInt, error: CInt) {
     return _pipe(unsafeFds.baseAddress, 0, 0)
 #elseif os(WASI)
     preconditionFailure("No pipes available on WebAssembly/WASI")
+#elseif os(Android)
+    return pipe(unsafeFds.baseAddress!)
```

Comment:
This worked with the last NDK 25c too, but I split this off for Android in case it might affect other platforms if I modified the line below.